File: src/ast/DynamicExceptionSpec.java
package ast;

import java.io.StringWriter;

public class DynamicExceptionSpec extends ParentNode {
    public DynamicExceptionSpec(BaseNode child) {
        super(NodeType.DynamicExceptionSpec, child);
    }

    @Override
    public void PrintLeft(StringWriter writer) {
        writer.write("throw(");
        child.Print(writer);
        writer.write(")");
    }
}

File: src/de/fearlesstobi/demangler/ast/DynamicExceptionSpec.kt
package de.fearlesstobi.demangler.ast

import java.io.StringWriter

class DynamicExceptionSpec(child: BaseNode?) : ParentNode(NodeType.DynamicExceptionSpec, child) {
    public override fun printLeft(writer: StringWriter) {
        writer.write("throw(")
        child!!.print(writer)
        writer.write(")")
    }
}